db/compaction/tiered_compaction_test.cc

```diff
@@ -1434,6 +1434,59 @@ TEST_F(PrecludeLastLevelTest, MigrationFromPreserveTimePartial) {
   Close();
 }
 
+TEST_F(PrecludeLastLevelTest, SmallPrecludeTime) {
+  const int kNumTrigger = 4;
+  const int kNumLevels = 7;
+  const int kNumKeys = 100;
+
+  Options options = CurrentOptions();
+  options.compaction_style = kCompactionStyleUniversal;
+  options.preclude_last_level_data_seconds = 60;
```

Comment:
I know preserve_internal_time_seconds is 0 by default. But it might be better to set here too to make the test more explicit.